VideoApi/VideoApi/Services/VirtualRoomService.cs

```diff
@@ -100,7 +101,11 @@ private async Task<ParticipantRoom> RetrieveJudicialRoomForConference(Guid confe
         {
             _logger.LogInformation("Creating a new interpreter room for conference {Conference}", conference.Id);
             var roomId = await CreateInterpreterRoom(conference.Id, roomType);
-            var ingestUrl = roomType == VirtualCourtRoomType.JudicialShared? null : $"{conference.IngestUrl}/{roomId}";
+            var ingestUrl = roomType == VirtualCourtRoomType.JudicialShared? null :
```

Comment:
that is not easy to read. can you please change this

Reply:
Make sense and amended.